addons/ai/CfgEventHandlers.hpp

```diff
@@ -0,0 +1,23 @@
+class Extended_PreStart_EventHandlers {
+    class ADDON {
+        init = QUOTE(call COMPILE_FILE(XEH_preStart));
+    };
+};
+
+class Extended_PreInit_EventHandlers {
+    class ADDON {
+        init = QUOTE(call COMPILE_FILE(XEH_preInit));
+    };
+};
+
+class Extended_PostInit_EventHandlers {
+    class ADDON {
+        init = QUOTE(call COMPILE_FILE(XEH_postInit));
+    };
+};
+
+class Extended_FiredNear_EventHandlers {
+    class CAManBase {
+        init = QUOTE(call COMPILE_FILE(mortarSupportFiredNear));
```

Comment:
My bad. I originally copied this example but it didn't work when I tested in game. Ill change and test again.